api/routes/index.js

```diff
@@ -1,4 +1,27 @@
-import express from 'express';
-const router = express.Router();
+export function register(server, options, next) {
+  /* Serve static files from public folder */
+  server.route({
```

Comment:
By that I mean the public stuff, the index route should stay here and the file should then in turn import child routes/endpoints.